test/crit/Makefile

```diff
@@ -1,18 +1,24 @@
 CC ?= gcc
+GO ?= go
 
 all: test clean
 
 test: gen-imgs
+	@echo "Running integration test"
+	@./crit-test
```

Comment:
It might be good to separate these tests, e.g., `crit-test-integration` and `crit-test-e2e`.
This would make it easier to debug the tests.